tests/functional/test_deployment_config.py

```diff
@@ -291,7 +291,7 @@ def test_drum_prediction_server_pps_response(
             input_dataset = resources.datasets(framework, problem)
 
             # do predictions
-            for endpoint in ["/predict/", "/predictions/"]:
+            for endpoint in ["/predict/", "/predictions/", "/invocations"]:
```

Comment:
Please extract this list out into conftest.

Reply:
Done